src/webhooks/pubsub/stalebot.test.ts

```diff
@@ -96,7 +96,7 @@ But! If you comment or otherwise update it, I will reset the clock, and if you r
             {
               ...issueInfo,
               labels: [WAITING_FOR_COMMUNITY_LABEL, WORK_IN_PROGRESS_LABEL],
-              pull_request: {},
+              merge_commit_sha: '12345',
```

Comment:
nit: could put a constant at the top of the file to make reading this a bit easier:
```
const FAKE_MERGE_COMMIT_SHA = '12345'
```

Reply:
sure